tests/presenters/test_list_drafts_presenter.py

```diff
@@ -21,7 +21,7 @@ def test(self) -> None:
             results=[
                 ListedDraft(
                     id=draft_id,
-                    creation_date=datetime.min,
+                    creation_date=datetime.now(),
```

Comment:
In the test module, it's in general preferable to use the `FakeDatetimeService` instead of the built-in `datetime`. See for example `tests/presenters/test_get_company_dashboard_presenter.py`. (But we ourselfs did not always use this consistently in the past)